src/booleans.js

```diff
@@ -1,59 +1,66 @@
 export const negate = (a) => {
-  // your code here
+  return !a;
 };
 
 export const both = (a, b) => {
-  // your code here
+  return (a && b);
 };
 
 export const either = (a, b) => {
-  // your code here
+  return (a || b)  || (b || a);
 };
 
 export const none = (a, b) => {
-  // your code here
+  return (!a && !b);
 };
 
 export const one = (a, b) => {
-  // your code here
+  return (!a && b) || (a && !b);
 };
 
 export const truthiness = (a) => {
-  // your code here
+ return !(!a);
 };
 
 export const isEqual = (a, b) => {
-  // your code here
+  return a == b;
 };
 
 export const isGreaterThan = (a, b) => {
-  // your code here
+  return a > b;
 };
 
 export const isLessThanOrEqualTo = (a, b) => {
-  // your code here
+  return a <= b;
 };
 
 export const isOdd = (a) => {
-  // your code here
+  return a % 2 == !0;
```

Comment:
Always use strict equality/inequality. Ideal solution here would be `return a % 2 !== 0`